backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/NormService.java

```diff
@@ -90,20 +92,28 @@ public Optional<String> updateNormXml(UpdateNormXmlUseCase.Query query)
   }
 
   @Override
-  public List<String> loadSpecificArticles(LoadSpecificArticleXmlFromNormUseCase.Query query) {
+  public List<String> loadSpecificArticlesXmlFromNorm(
```

Comment:
I kept it like this because that was the original behavior, but you're right it's not very consistent. Will update it to return a 404 when no articles are found